### PR TITLE
Bug: fix slice init length

### PR DESCRIPTION
**Pull request by dufucun**
## Proposed commit message

fix slice init length

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by dufucun** [comment]
> ❌ Author of the following commits did not sign a [Contributor Agreement](https://www.elastic.co/contributor-agreement): [14cb6ed](https://github.com/elastic/beats/commit/14cb6ed75e90575b4fa86fe74dc48228f03357ec)
> 
> Please, read and sign the above mentioned agreement if you want to contribute to this project

signed

File: libbeat/processors/ratelimit/limit.go
Comment (mauri870) [inline libbeat/processors/ratelimit/limit.go:1]: Seems like strconv.ParseFloat(valueStr, 8) is invalid, looking at the code for [strconv.ParseFloat](https://cs.opensource.google/go/go/+/refs/tags/go1.23.0:src/strconv/atof.go;l=704-708) what we meant was bit size 64. Can you fix it? Thanks.

**Comment by mauri870** [comment]
Thanks!

File: libbeat/processors/ratelimit/limit.go
Comment (mauri870) [reply libbeat/processors/ratelimit/limit.go:1]: Surprisingly, according to the documentation any bitSize different than 32 or 64 is undefined behavior https://pkg.go.dev/strconv#ParseFloat.

**Review by mauri870 — APPROVED** [review]

**Comment by dufucun** [comment]
Thanks for your review!

**Comment by pierrehilbert** [comment]
run docs-build

File: libbeat/processors/ratelimit/limit.go
Comment (mauri870) [reply libbeat/processors/ratelimit/limit.go:1]: You can ignore the linter error, I filled https://github.com/elastic/beats/pull/40753 for a fix.

Thanks!

**Comment by mauri870** [comment]
@ycombinator The linter error should be fixed once https://github.com/elastic/beats/pull/40753 is merged.

**Comment by ycombinator** [comment]
buildkite test this

**Comment by ycombinator** [comment]
buildkite test this

**Comment by pierrehilbert** [comment]
run docs-build

**Comment by pierrehilbert** [comment]
run docs-build

**Comment by mauri870** [comment]
Would be good to get this merged. The changes are very simple and logically correct. The CI errors  in the previous builds appear to be unrelated with the changes.

**Comment by pierrehilbert** [comment]
run docs-build

**Comment by dufucun** [comment]
Thanks

**Review by faec — APPROVED** [review]

**Review by AndersonQ — APPROVED** [review]

**Comment by pierrehilbert** [comment]
run docs-build

**Comment by pierrehilbert** [comment]
run docs-build

**Pull request merged**